### ISSUE TITLE
Klipper ignoring Slicer settings.

### ISSUE THREAD

**Issue by sinicolau6** [KevinOConnor/klipper#2217]
[klippy.log](https://github.com/KevinOConnor/klipper/files/3889241/klippy.log)

Basically, ive got everything running finally, but when i wanted to print out a Tunning Tower for pressure_advance, my printer started doing my G code macro for Starting the Print, then it would prime the nozzle with a solid line, then it would move to start printing, but it wouldnt extrude any filament. So i started debugging and it seems like klipper is ignoring any kind of setting sent from the Slicer of choice to the gcode exported file, except the G code macro that i mentioned earlier and movement steps. Anything else i figured is ignored. Here is my printer.cfg

[stepper_x]
step_pin: PB13
dir_pin: !PB12
enable_pin: !PB14
step_distance: .0125
endstop_pin: ^PC0
position_endstop: 0
position_max: 235
homing_speed: 50

[tmc2209 stepper_x]
uart_pin: PB15
microsteps: 16
run_current: 0.580
hold_current: 0.500
stealthchop_threshold: 250

[stepper_y]
step_pin: PB10
dir_pin: !PB2
enable_pin: !PB11
step_distance: .0125
endstop_pin: ^PC1
position_endstop: 0
position_max: 235
homing_speed: 50

[tmc2209 stepper_y]
uart_pin: PC6
microsteps: 16
run_current: 0.580
hold_current: 0.500
stealthchop_threshold: 250

[stepper_z]
step_pin: PB0
dir_pin: PC5
enable_pin: !PB1
step_distance: .0025
endstop_pin: probe:z_virtual_endstop
#position_endstop: 0.0
position_max: 250
position_min: -1.5

[tmc2209 stepper_z]
uart_pin: PC10
microsteps: 16
run_current: 0.580
hold_current: 0.500
stealthchop_threshold: 5

[extruder]
step_pin: PB3
dir_pin: !PB4
enable_pin: !PD2
step_distance: 0.01039965
nozzle_diameter: 0.400
filament_diameter: 1.750
heater_pin: PC8
sensor_type: EPCOS 100K B57560G104F
sensor_pin: PA0
#control: pid
#pid_Kp: 29.16
#pid_Ki: 2.93
#pid_Kd: 72.47
min_temp: 0
max_temp: 250

[tmc2209 extruder]
uart_pin: PC11
microsteps: 16
run_current: 0.650
hold_current: 0.500
stealthchop_threshold: 5

[heater_bed]
heater_pin: PC9
sensor_type: ATC Semitec 104GT-2
sensor_pin: PC3
#control: pid
#pid_Kp: 54.027
#pid_Ki: 0.770
#pid_Kd: 948.182
min_temp: 0
max_temp: 130

[fan]
pin: PA8

[mcu]
serial: /dev/serial/by-id/usb-Klipper_Klipper_firmware_12345-if00

[printer]
kinematics: cartesian
max_velocity: 300
max_accel: 3000
max_z_velocity: 5
max_z_accel: 100

[static_digital_output usb_pullup_enable]
pins: !PC13

[board_pins]
aliases:
    # EXP1 header
    EXP1_1=PB5, EXP1_3=PA9,   EXP1_5=PA10, EXP1_7=PB8, EXP1_9=<GND>,
    EXP1_2=PB6, EXP1_4=<RST>, EXP1_6=PB9,  EXP1_8=PB7, EXP1_10=<5V>

[display]
lcd_type: st7920
cs_pin: EXP1_7
sclk_pin: EXP1_6
sid_pin: EXP1_8
encoder_pins: ^EXP1_5, ^EXP1_3
click_pin: ^!EXP1_2

[output_pin beeper]
pin: EXP1_1

[bltouch]
control_pin:PA1
sensor_pin: PC2
x_offset: -42.6
y_offset: -16.4
#z_offset: 0
pin_move_time: 0.4

[bed_mesh]
speed: 100
horizontal_move_z: 10
min_point: 90, 30
max_point: 235,235
probe_count: 3, 3
fade_start: 1.0
mesh_pps: 2,2
relative_reference_index: 4

[gcode_macro G29]
gcode:
	G28
	G1 Z10 F600
	BED_MESH_CALIBRATE
	BED_MESH_OUTPUT

[gcode_macro START_PRINT]
gcode:
  M140 S60
  M204 P500.00 R1000.00 T500.00 
  M220 S100 
  M221 S100 
  G29
  M109 S200
  M190 S60
  G92 E0 
  G1 Z10 F3000 
  G1 X10.1 Y20 Z0.28 F5000.0 
  G1 X10.1 Y200.0 Z0.28 F1500.0 E15 
  G1 X10.4 Y200.0 Z0.28 F5000.0 
  G1 X10.4 Y20 Z0.28 F1500.0 E30 
  G92 E0 
  G1 Z2.0 F3000

[safe_z_home]
home_xy_position: 155,120
speed: 80.0
z_hop: 10.0
z_hop_speed: 10.0

   	

#*# <---------------------- SAVE_CONFIG ---------------------->
#*# DO NOT EDIT THIS BLOCK OR BELOW. The contents are auto-generated.
#*#
#*# [bed_mesh default]
#*# points =
#*# 	0.135000, 0.180000, 0.240000
#*# 	-0.002500, -0.015000, -0.045000
#*# 	0.135000, 0.092500, 0.052500
#*# x_count = 3
#*# y_count = 3
#*# min_x = 90.0
#*# max_x = 235.0
#*# min_y = 30.0
#*# max_y = 235.0
#*# x_offset = -42.6
#*# y_offset = -16.4
#*# mesh_x_pps = 2
#*# mesh_y_pps = 2
#*# algo = lagrange
#*# tension = 0.2
#*#
#*# [extruder]
#*# control = pid
#*# pid_kp = 28.858
#*# pid_ki = 1.924
#*# pid_kd = 108.217
#*#
#*# [heater_bed]
#*# control = pid
#*# pid_kp = 59.962
#*# pid_ki = 1.285
#*# pid_kd = 699.312
#*#
#*# [bltouch]
#*# z_offset = 3.980

I tried this with S3D and PrusaSlic3r with the same results.

The bed is calibrated, and so is the nozzle and the probe with the coresponding offsets, i checked and double checked that so  many times. Im at a loss right now and i need anybody's help in figuring this out.

There are no hardware issues, i made sure of that too. Its software wise somehow.

Basically, the G code Macro "START_PRINT" works perfectly, after that stops, the extruder still moves but it doesnt push  any filament through.

**Comment by sinicolau6** [comment]
I hate myself. Apparently, if you have Octolapse in test mode it wouldnt extrude. That was the fix.
